server/controllers/accommodationController.js

```diff
@@ -0,0 +1,107 @@
+import AccommodationHelper from '../helpers/accommodationHelper';
+
+/**
+ * This class contains all methods
+ * required to handle all
+ * accommodation-related operations.
+ */
+class Accommodation {
+  /**
+   * This method handles the register request.
```

Comment:
Use **This method registers accommodation** instead.

Reply:
Feedback implemented.